### PR TITLE
Upgrade to Jazzy/Ionic

### PR DESCRIPTION
**Pull request by MarqRazz**

### CODE REVIEW

**Comment by MarqRazz** [comment]
I'm still getting this error when starting Gazebo. Any ideas on what is wrong with the gripper configuration?
```
[gazebo-3] [Err] [Physics.cc:1773] Attempting to create a mimic constraint for joint [mimic_gripper_joint] but the chosen physics engine does not support mimic constraints, so no constraint will be created.
```

**Comment by MarqRazz** [comment]
Other warnings and errors we should fix here...
```
[gazebo-3] Error parsing the limits for the interface: position from the tags [min: '0.0' and max: '{pi}'] within ros2_control tag inside the URDF. Skipping it
[gazebo-3] Error parsing the limits for the interface: position from the tags [min: '-{pi}' and max: '{pi}'] within ros2_control tag inside the URDF. Skipping it
```
and 
```
[gazebo-3] [gz_ros_control.resource_manager 1732902976.597228392]: Importing state interfaces for the hardware 'piper_control' returned no state interfaces.
```

**Comment by bmagyar** [comment]
The `pi` value doesn't get rolled out. I think it is used but not defined in the URDF.

File: piper_bringup/package.xml [hunk 1]
```diff
@@ -11,8 +11,8 @@
 
   <exec_depend>rviz2</exec_depend>
   <exec_depend>controller_manager</exec_depend>
-  <exec_depend>ign_ros2_control</exec_depend>
-  <exec_depend>gripper_action_controller</exec_depend>
+  <exec_depend>gz_ros2_control</exec_depend>
+  <exec_depend>gripper_controllers</exec_depend>
```
Comment (bmagyar) [inline piper_bringup/package.xml]:
```suggestion
  <exec_depend>parallel_gripper_controller</exec_depend>
```
how about this instead?

**Comment by MarqRazz** [comment]
Okay I fixed most of the errors and the gripper now works in Gazebo :1st_place_medal:. I had to switch us to [bullet featherstone physics](https://github.com/ros-controls/gz_ros2_control/issues/340) because the ParallelGripper controller does not appear to automatically support mimic joints like the [ForwardCommandController](https://github.com/ros-controls/gz_ros2_control/blob/07a2d3b2519e36e3b86194b286869f292e4e6782/gz_ros2_control_demos/config/gripper_controller_position.yaml#L10) does in the examples.

piper_gripper

We still get this warning...
```
[gazebo-3] [gz_ros_control.resource_manager 1732980292.597212909]: Importing state interfaces for the hardware 'piper_control' returned no state interfaces.
```

Looks like we are falling back to this base class function for [import_state_interfaces](https://github.com/ros-controls/ros2_control/blob/2b22d0cdf28412fc4fde7200c367473dfba754af/hardware_interface/src/resource_manager.cpp#L614-L617)

**Pull request merged**